### ISSUE TITLE
Option to toggle between celsius and fahrenheit.

### ISSUE THREAD

**Issue by krtkush** [krtkush/MarsExplorer#27]
A simple toggle on the home screen that can enable user to select temperature unit preference. The preference becomes the default unit for that user.

**Comment by utkarshbhatt12** [comment]
Where do you think the toggle should be? 
![screenshot_20170228-212009](https://cloud.githubusercontent.com/assets/10895594/23412625/21ec9fd8-fdfc-11e6-8e61-821afd442654.jpg)

I think the circle would be a good place.

**Comment by krtkush** [comment]
@utkarshbhatt12 I can't see the screenshot; the link does not seem to be working.

**Comment by utkarshbhatt12** [comment]
The screenshot isn't uploading here. I don't know why. Here's a mirror: http://oi63.tinypic.com/ay1vtl.jpg

**Comment by krtkush** [comment]
Yep! The circle is where I plan to put the toggle. 

Why do you have that arrow pointing to "About", though?

**Comment by utkarshbhatt12** [comment]
Some apps have dedicated activity for Settings. I was giving out the idea that we could do that here too. This would be the best approach if you ever decide to add more toggles and stuff.

**Comment by krtkush** [comment]
@utkarshbhatt12 That's true. I'll consider a dedicated setting activity if more such toggles come up.
For the next version I'll keep the toggle on the weather card itself.
